### ISSUE TITLE
Some questions about the semantic part and training results

### ISSUE THREAD

**Issue by Luyunqx** [zju3dv/street_gaussians#68]
Hi Street-Gaussian authors
Thank you for releasing your wonderful work.I noticed in your paper that you mentioned using semantic information as part of the loss function, but when I checked the source code, it seems like this feature is disabled. Also, I couldn’t find any semantic ground truth data in the example dataset.
The evaluation metrics from the training results I got using your source code didn’t match the high values you mentioned in the paper. Specifically, for the Waymo dataset, the PSNR for 002 is only 29.64, for 031 it’s 27.74, and for 111 it’s 31.70. I’m wondering if this could be an issue with my hardware setup. I’m using an Nvidia RTX4090 with 24GB of VRAM and a 13th Gen Intel Core i7-13700K CPU. Could you shed some light on whether this might be a hardware-related issue or if there’s something else I should be looking into?
